control-station/src/views/Dynamics/Dynamics.tsx

```diff
@@ -11,22 +11,26 @@ function Dynamics() {
 	return (
 		<div className="dynamics">
 			<div className="dynamics-sensorbox">
-				<SensorBox title="Roll" value={gyroscope.roll} />
-				<SensorBox title="Pitch" value={gyroscope.pitch} />
+				<SensorBox title="Roll" value={Math.round(gyroscope.roll * 100) / 100} />
+				<SensorBox title="Pitch" value={Math.round(gyroscope.pitch * 100) / 100} />
 			</div>
 			<div className="dynamics-pictures">
 				<div className="pod-picture-container">
 					<img
 						src={FrontPod}
 						alt="front view of the pod"
-						style={{ transform: `rotate(${gyroscope.roll}deg)` }}
+						style={{
+							transform: `rotate(${Math.round(gyroscope.roll * 100) / 100}deg)`,
+						}}
```

Comment:
Thought: I can see how we want format the display value, but does rounding matter for the transformation?